nixos/doc/manual/development/writing-nixos-tests.xml

```diff
@@ -108,7 +108,7 @@ xlink:href="https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/virtualis
 <programlisting>
 $machine->start;
 $machine->waitForUnit("default.target");
-$machine->succeed("uname") =~ /Linux/;
+die unless $machine->succeed("uname") =~ /Linux/;
```

Comment:
I don't love this change because it isn't actually something we ever _do_ in the tests (there are no `die unless`s) but I do agree we shouldn't have the comparison just be ignored.

Reply:
Should I change it to `$machine->succeed('[[ $(uname) =~ Linux ]]')` then?

Reply:
Nah, let's take it as-is. Thank you!